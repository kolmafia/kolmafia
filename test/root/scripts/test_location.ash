// snarfblat locations have an id
print("Snarfblat location has an id: " + ($location[Noob Cave].id == 240));
print("Snarfblat location can be converted to int: " + ($location[The Dire Warren].to_int() == 92));

// non snarfblat locations
print("Non-snarfblat locationas have id of -1: " + ($location[Summoning Chamber].id == -1));
print("Non-snarfblat locationas have int value of -1: " + ($location[The Lower Chambers].to_int() == -1));

// can adventure
print("Location none cannot be adventured in: " + ($location[none].can_adventure() == false));