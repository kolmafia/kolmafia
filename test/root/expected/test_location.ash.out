Snarfblat location has an id: true
Snarfblat location can be converted to int: true
Non-snarfblat locationas have id of -1: true
Non-snarfblat locationas have int value of -1: true
Location none cannot be adventured in: true